### PR TITLE
Removing the public interface that enforce dependancy on System.Confi…

### LINKED ISSUE #1007: Remove all Instances of System.Configuration
From email thread:

> In .NETCore we don’t recommend the use of app.config (none of the framework types honor its settings) nor does machine.config exist. 

> App.config doesn’t flow into AppContext since CoreCLR doesn’t read app.config.  AppContext still exists and you can access the values via the API.

### PR DESCRIPTION
**Pull request by dreddy-work**
…guration.Manager package.

We still depend on this package for 'ToolstripSettings'. We will revisit this dependancy
when we have solution for replacing/redesigning the way we read those settings.
fixes #1007

### CODE REVIEW

File: src/System.Windows.Forms/src/System/Windows/Forms/NativeWindow.cs [hunk 1]
```diff
@@ -304,31 +304,13 @@ private static int WndProcFlags
...
-                                        Debug.WriteLineIf(WndProcChoice.TraceVerbose, "After config check 0x" + intWndProcFlags.ToString("X", CultureInfo.InvariantCulture));
-                                    }
-                                    else
-                                    {
-                                        intWndProcFlags |= UseDebuggableWndProc;
-                                    }
-                                }
-                            }
+                        else {
+                            // Reading Framework key in Netcore/5.0 doesn't make sense. This path seems to be used to override the
```
Comment (zsd4yr) [inline src/System.Windows.Forms/src/System/Windows/Forms/NativeWindow.cs]: Do we need any of this comment anymore?

File: src/System.Windows.Forms/src/System/Windows/Forms/WindowsFormsSectionHandler.cs [hunk 2]
```diff
@@ -2,79 +2,25 @@
...
-        protected override ConfigurationPropertyCollection Properties
-        {
-            get
-            {
-                return EnsureStaticPropertyBag();
-            }
-        }
-
-        [ConfigurationProperty("jitDebugging", DefaultValue = JitDebuggingDefault)]
         public bool JitDebugging
```
Comment (zsd4yr) [inline src/System.Windows.Forms/src/System/Windows/Forms/WindowsFormsSectionHandler.cs]: Can we just remove this class and all references to it then?

File: src/System.Windows.Forms/src/System/Windows/Forms/NativeWindow.cs (hunk 1 above)
Comment (dreddy-work) [reply src/System.Windows.Forms/src/System/Windows/Forms/NativeWindow.cs]: This comment is to keep history on why we are removing it and what  we may need to do later if we have to bring it in, in a different way.

File: src/System.Windows.Forms/src/System/Windows/Forms/WindowsFormsSectionHandler.cs (hunk 2 above)
Comment (dreddy-work) [reply src/System.Windows.Forms/src/System/Windows/Forms/WindowsFormsSectionHandler.cs]: Having skeleton will throw a meaningful exception and also may unblock if there are any compile time dependency ( i doubt)

File: src/System.Windows.Forms/src/System/Windows/Forms/WindowsFormsSectionHandler.cs [hunk 3]
```diff
@@ -2,79 +2,25 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode", Scope = "member", Target = "System.Windows.Forms.WindowsFormsSectionHandler..ctor()")]
+namespace System.Windows.Forms {
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/WindowsFormsSectionHandler.cs]: `{` are not correctly position - they should stay on a new line

File: src/System.Windows.Forms/src/System/Windows/Forms/NativeWindow.cs [hunk 4]
```diff
@@ -304,31 +304,13 @@ private static int WndProcFlags
...
+                            // default behaviour after applications deployed ( otherwise, Developer/user can set this flag
+                            // via Application.SetUnhandledExceptionModeInternal(..).
+                            // Disablign this feature from .NET core 3.0 release. Would need to redesign if there are customer requests on this.
```
Comment (hughbe) [inline src/System.Windows.Forms/src/System/Windows/Forms/NativeWindow.cs]: nit: typo

**Pull request merged**